packages/code-snippet-block/src/settings.ts

```diff
@@ -0,0 +1,337 @@
+                },
+                {
+                    value: 'materialLighter',
+                    label: 'Material Lighter',
+                },
+                {
+                    value: 'materialPalenight',
+                    label: 'Material Palenight',
+                },
+                {
+                    value: 'oneDark',
+                    label: 'One Dark',
+                },
+                {
+                    value: 'oneLight',
+                    label: 'One Light',
+                },
+                {
+                    value: 'base16Dark',
+                    label: 'Base16 Dark',
+                },
+                {
+                    value: 'base16Light',
+                    label: 'Base16 Light',
+                },
+                {
+                    value: 'cobalt',
+                    label: 'Cobalt',
+                },
+            ],
+        },
+        {
+            id: 'withBorder',
+            type: 'switch',
+            defaultValue: true,
+            label: 'Border',
+        },
+        {
+            id: 'border',
+            type: 'multiInput',
+            lastItemFullWidth: true,
+            show: (bundle: Bundle) => bundle.getBlock('withBorder')?.value === true,
+            onChange: (bundle: Bundle): void => {
+                appendUnit(bundle, 'lineWidth');
+            },
+            blocks: [
+                {
+                    id: 'lineStyle',
+                    type: 'dropdown',
+                    defaultValue: 'solid',
+                    choices: [
+                        {
+                            value: 'none',
+                            label: 'None',
+                        },
+                        {
+                            value: 'dotted',
+                            label: 'Dotted',
+                        },
+                        {
+                            value: 'dashed',
+                            label: 'Dashed',
+                        },
+                        {
+                            value: 'solid',
+                            label: 'Solid',
+                        },
+                        {
+                            value: 'double',
+                            label: 'Double',
+                        },
+                    ],
+                },
+                {
+                    id: 'lineWidth',
+                    type: 'input',
+                    placeholder: 'e.g. 2px',
+                    rules: [numericalOrPixelRule],
+                },
+                {
+                    id: 'borderColor',
+                    type: 'colorInput',
+                    defaultValue: BORDER_COLOR_DEFAULT_VALUE,
+                },
+            ],
+            layout: MultiInputLayout.Columns,
+        },
+        /** Commented out because of the issue https://github.com/Frontify/guideline-blocks/pull/141#issuecomment-1231385790 */
+        // {
```

Comment:
Dead code

Reply:
same as above.

Reply:
The feature is not used and the code is commented out without any plans to solve it soon, this is dead code ;)

Reply:
But it will be used again. Why rewrite stuff? It is relevant.

Reply:
If you have a strong opinion on this, I will remote it.

Reply:
If you plan to fix it next sprint, fine for me to stay
If no ETA, you can extract the commented code to a new branch so we can merge this one